core-java-8/src/test/java/com/baeldung/shufflingcollections/ShufflingCollectionsUnitTest.java

```diff
@@ -0,0 +1,70 @@
+package com.baeldung.shufflingcollections;
+
+import org.junit.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ShufflingCollectionsUnitTest {
+
+    @Test
+    public void whenShufflingList_thenListIsShuffled() {
+        List<String> students = Arrays.asList("Foo", "Bar", "Baz", "Qux");
+
+        System.out.println("List before shuffling:");
+        System.out.println(students);
+
+        Collections.shuffle(students);
+        System.out.println("List after shuffling:");
+        System.out.println(students);
+    }
+
+    @Test
+    public void whenShufflingMapKeys_thenValuesAreShuffled() {
+        Map<Integer, String> studentsById = new HashMap<>();
+        studentsById.put(1, "Foo");
+        studentsById.put(2, "Bar");
+        studentsById.put(3, "Baz");
+        studentsById.put(4, "Qux");
+
+        System.out.println("Students before shuffling:");
+        System.out.println(studentsById.values());
+
+        List<Integer> shuffledStudentIds = new ArrayList<>(studentsById.keySet());
+        Collections.shuffle(shuffledStudentIds);
+
+        List<String> shuffledStudents = shuffledStudentIds.stream()
+                .map(id -> studentsById.get(id))
```

Comment:
indent for lines 39-40 need to be only two spaces over from line 38, instead of 8 spaces

Reply:
Done.